hail/python/hail/methods/statgen.py

```diff
@@ -159,7 +159,7 @@ def impute_sex(call, aaf_threshold=0.0, include_par=False, female_threshold=0.2,
 
     5. For each row and column with a non-missing genotype call, :math:`E`, the
        expected number of homozygotes (from population AAF), is computed as
-       :math:`1.0 - (2.0*maf*(1.0-maf))`.
+       :math:`1.0 - (2.0*\mathrm{maf}*(1.0-\mathrm{maf}))`.
```

Comment:
Why is mathrm preferred here, over italic?

Reply:
The issue is that in math mode in TeX, spaces are mostly irrelevant. So something like ``maf`` is interpreted as three variable names, not one identifier. For example, you would typeset a polynomial as ``$f(x, y) = 5x^2 + 10xy$``, where ``xy`` is x times y, not something named "xy".  When you want a single identifier, this can cause the spacing to be off. It's standard to use ``\mathrm`` or something similar, and we do this other places in the docs, so this is consistent.

Default role is a sphinx thing, but this is being processed by KaTeX.

Reply:
> maf is interpreted as three variable names

Ah, thanks.

> Default role is a sphinx thing, but this is being processed by KaTeX.

Got it, hands off to KaTex when it encounters .. math:: or :math: 

Thanks!